src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc

```diff
@@ -31,6 +31,17 @@ class UniqueIdFromJByteArray {
   }
 };
 
+inline void ThrowRayExceptionIfNotOK(JNIEnv *env, const ray::Status &status,
+                                     const std::string &message) {
+  if (!status.ok()) {
+    jclass exception_class = env->FindClass("org/ray/api/exception/RayException");
+    if (exception_class == 0) {
+      exception_class = env->FindClass("java/lang/RuntimeException");
```

Comment:
why is this needed?

Reply:
When I implemented this test at the first time, I used `.get()` but I still found there are core dump files. This code is used to debug to see whether the `RayException` class is found or not. I found the crash is caused by Plasma Client. I will later remove the `ARROW_CHECK` out of plasma client JNI. I will remove these 3 lines.